src/psyclone/dynamo0p3.py

```diff
@@ -1301,18 +1234,20 @@ def __init__(self, node):
         extent_names = []
         for call in self._calls:
             for arg in call.arguments.args:
-                if arg.stencil:
-                    # Check for the existence of arg.extent here as in
-                    # the future we plan to support kernels which
-                    # specify the value of extent in metadata. If this
-                    # is the case then an extent argument is not
-                    # required.
-                    if not arg.stencil.extent:
-                        if not arg.stencil.extent_arg.is_literal():
-                            if arg.stencil.extent_arg.text not in extent_names:
-                                extent_names.append(
-                                    arg.stencil.extent_arg.text)
-                                self._unique_extent_args.append(arg)
+                if not arg.stencil:
+                    continue
+                # Check for the existence of arg.extent here as in
+                # the future we plan to support kernels which
+                # specify the value of extent in metadata. If this
+                # is the case then an extent argument is not
+                # required.
+                if arg.stencil.extent:
+                    continue
```

Comment:
I tried for around one hour, then noticed in dynamo0p3, around line 8161
```
                    raise GenerationError("extent metadata not yet supported")
                    # if supported we would add the following
                    # line. However, note there is currently no setter
                    # for extent in DynStencil so this would need to
                    # be added.  stencil.extent =
                    # dyn_argument.descriptor.stencil['extent']
```
So, with this not being supported, I think I just run into problems, create crappy test cases etc. So I reverted the changes to this loop, and disabled the warning.

Reply:
Thanks Joerg. Sorry it cost you so much time.